### ISSUE TITLE
Setting a MIDI device to play a different instrument makes it play both the old and new instruments

### ISSUE THREAD

**Issue by WarpedWartWars** [jmoenig/Snap#3071]
https://forum.snap.berkeley.edu/t/how-could-i-connect-snap-to-a-midi-device/11510/7
> [T]here's a bug where if you run this:
![image](https://user-images.githubusercontent.com/91039320/180101484-cb75136b-32f6-40e9-bec6-a90dd044c2f5.png)
it makes it sound like a piano and a guitar, combined.

**Comment by jmoenig** [comment]
yeah, see my comment(s) to https://github.com/jmoenig/Snap/issues/3072

**Comment by WarpedWartWars** [comment]
I've fixed this in [a branch of my fork](https://github.com/WarpedWartWars/Snap/tree/tunescope-fix). You can try it [here](https://warpedwartwars.github.io/Snap/snap.html). (I've also fixed the `Note`/`_Note` thing by renaming references to `Note` in `webmidi.iife.js` to `_Note`)
